JenkinsFile

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/groovy
+
+node {
+    def root = pwd()
+    def mvn = tool 'M3'
+    def fortifyBuild = "bfalg-ndwi-${env.BUILD_NUMBER}"
+    def sonar = tool 'SonarQube Scanner 3.0'
+    def projectId = "venicegeo:beachfront:bfalg-ndwi"
+    def appvers = ""
+    def appName = ""
+	
+    stage("Setup") {
+        deleteDir()
+	sh "mkdir bfalg-ndwi"
+	dir("bfalg-ndwi") {
+            git url: "${env.GIT_URL}", branch: "${env.GIT_BRANCH}"
```

Comment:
This works in .io, but in .mil it will fail because we have to use credentials.

Something like this is what we need:
```
  stage('Setup') {                                                              
    deleteDir()                                                                 
    // github is open gitlab requires credentials to clone                      
    if(env.GITLAB_CREDS) {                                                      
      git url: "${env.GIT_URL}", branch: "${env.GIT_BRANCH}", credentialsId: "${env.GITLAB_CREDS}"
    } else {                                                                    
      git url: "${env.GIT_URL}", branch: "${env.GIT_BRANCH}"                    
    }                                                                           
  } 
```

Reply:
Agreed